### DISCUSSION TITLE
Takara Smarter Seq

### DISCUSSION THREAD

**Discussion by bshim181** [milaboratory/mixcr#1792] [Q&A] [answered]
Hello, 

I have tried running the Takara Smarter Seq through MiXCR's preset and but struggling to get sufficient clone counts due to high proportion of reads getting discarded due to failed mapping in assembly. 

Alignment itself is not optimal, but I am seeing approximately 60% of reads aligning. 

<img width="1230" alt="Screenshot 2024-09-13 at 12 02 03 PM" src="https://github.com/user-attachments/assets/b6287d02-88e6-4f22-8242-e0fc4cb27ee8">

Reads used in clonotyping are only showing 6% with 40% of the reads dropped due to failed mapping. 
What might be the cause for this? Is there additional parameters required for the Takara smarter seq preset to process the data correctly?

<img width="819" alt="Screenshot 2024-09-13 at 12 00 48 PM" src="https://github.com/user-attachments/assets/3b1e4482-f0a6-4e2e-a62f-c156dc38abe7">

**Comment by mizraelson** [comment]
It seems like the sequencing quality is low and that is why MiXCR filters out those reads.

**Reply by bshim181** [reply]
when you say sequencing quality, do you mean Phred quality scores for each bases are low? is there a way to save those reads?

**Reply by mizraelson** [reply, marked as answer]
Yes, Phred quality. If needed, you can disable all thresholds by adding:

```
-Massemble.cloneAssemblerParameters.minimalQuality=0 \
-Massemble.cloneAssemblerParameters.badQualityThreshold=0 \
```
